src/swsssdk/configdb.py

```diff
@@ -202,8 +220,8 @@ def get_table(self, table):
                 pass    #Ignore non table-formated redis entries
         return data
 
-    def set_config(self, data):
-        """Write multiple tables into config db. 
+    def mod_config(self, data):
+        """Write multiple tables into config db. Existing entries in the db are kept.
```

Comment:
It's not "Existing entries are kept" (if existing entry occurs in `data` it will be modified), but "Untouched entries and untouched attributes of existing entries are kept".

Reply:
how about "extra entries/attributes in the db are kept"?